easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-v1612+-cleanup.patch

```diff
@@ -0,0 +1,479 @@
+--- OpenFOAM-v1612+.orig/applications/utilities/mesh/manipulation/setSet/Allwmake	2016-12-23 15:22:59.000000000 +0100
```

Comment:
please include a brief description of the patch above this line and mention the author

any lines above this line are comments